testsuite/MDAnalysisTests/analysis/test_encore.py

```diff
@@ -429,7 +429,8 @@ def ens2(self, ens2_template):
             ens2_template.filename,
             ens2_template.trajectory.timeseries(order='fac'),
             format=mda.coordinates.memory.MemoryReader)
-    
+
+
```

Comment:
I'm surprised that pep8speaks didn't complain here, it should be one empty lines between class methods.
```suggestion
```

Reply:
Ok I'm working on it, I just didn't add more tests cause It was mentioned that the starter issues are limited, so I didn't want to consume it. I will add a couple of tests to cover the whole Cluster method 🙌

Reply:
It's completely up to you. If you want to move towards a non-starter issue that's perfectly fine too. I'm happy to go ahead with merging this as it is.

Reply:
I have added more tests to cover the whole Cluster class + I fixed some typo errors. I'd be happy if you guided me to any improvements need to be done before merging